src/java.base/share/classes/jdk/internal/foreign/ConfinedSession.java

```diff
@@ -32,8 +32,8 @@
 import jdk.internal.vm.annotation.ForceInline;
 
 /**
- * A confined session, which features an owner thread. The liveness check features an additional
- * confinement check - that is, calling any operation on this session from a thread other than the
+ * A confined scope, which features an owner thread. The liveness check features an additional
```

Comment:
This should be left as session (since we have not changed the name of the class)